README.md

```diff
@@ -36,8 +36,28 @@ Start contributing to SwiftSyntax see [this guide](CONTRIBUTING.md) for more inf
 ## Bazel
 
 SwiftSyntax provides an experimental [Bazel](https://bazel.build) build configuration, maintained by Keith Smiley. 
-To use it you can pull the source archive from the relevant release tag
-into your `WORKSPACE` and depend on the libraries you need from the
+To use it, you can pull the source archive from the relevant release tag
+into your `MODULE.bazel` file (preferred and recommended) with `bazel_dep`. Bzlmod support was added starting release of `509.0.0` and above. All available versions can be found in the [Bazel Central Registry](https://registry.bazel.build/modules/swift-syntax)
+
+```python3
+bazel_dep(
+    name = "swift-syntax",
+    version = "600.0.1",
+)
```

Comment:
```suggestion
bazel_dep(name = "swift-syntax", version = "600.0.1")
```

seems to me like the community is generally preferring using this as a single line